src/components/bulkDataFiles/BulkDataFiles/BulkDataFiles.js

```diff
@@ -120,6 +142,7 @@ const mapDispatchToProps = (dispatch) => {
     loadBulkDataFilesDispatcher: () => dispatch(loadBulkDataFiles()),
     setApiErrorDispatcher: (api, state, errorMessage) => dispatch(setApiError(api, state, errorMessage)),
     updateBulkDataFilesDispatcher: (bulkDataFiles) => dispatch(updateBulkDataFiles(bulkDataFiles)),
+    authApiStatusDispatcher: (status) => dispatch(authApiStatus(status)),

```

Comment:
This function takes zero arguments:
```suggestion
    authApiStatusDispatcher: () => dispatch(authApiStatus()),
```